frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -493,6 +493,7 @@
   "Rate of transmitted packets dropped": "Rate of transmitted packets dropped",
   "Average Container bandwidth by Pod: received": "Average Container bandwidth by Pod: received",
   "Average Container bandwidth by Pod: transmitted": "Average Container bandwidth by Pod: transmitted",
+  "{preText}<1>{opt}</1>{postText}": "{preText}<1>{opt}</1>{postText}",
```

Comment:
I don't think this would work since the 'and' would still end up not being translated.